### ISSUE TITLE
Getting Started examples seem incomplete?

### ISSUE THREAD

**Issue by cfjedimaster** [javve/list.js#761]
When looking at the Getting Started docs (https://listjs.com/docs/), if one were to use the first example, it doesn't seem to actually do anything. You need to add the input fields for filtering and sorting. This is obvious if you read the home page, but shouldn't the example actually include this? By itself it would do nothing, right? (That's what I saw when using it as is.)

**Comment by Mateen-Aslam** [comment]
check this: https://listjs.com/examples/data-attributes-custom-attributes/
else show code

**Comment by cfjedimaster** [comment]
I'm sorry but I don't understand your comment. I'm not saying I don't know how to use List.js. I'm pointing out how the docs for the getting started page could possibly be improved.

**Comment by Mateen-Aslam** [comment]
I`m sorry. would you please see his issue: https://github.com/javve/list.js/issues/764

**Comment by cfjedimaster** [comment]
How does that issue have anything to do with what I asked? The issue you linked to has nothing to do with what I brought up here.

**Comment by Mateen-Aslam** [comment]
Yes you are right at first, at first I also suck by reading the home page. What someorn can do is at first explain with example about what listjs do, then maybe show the page that redirect to the other example.
1- with existing html
2-  Index existing list and then add
.
.
.
and so one
what my opinion, now what yours ?
